scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -1141,7 +1141,7 @@ object FormatWriter {
   // "slc" stands for single-line comment
   private val slcDelim = Pattern.compile("\\h++")
   // "mlc" stands for multi-line comment
-  private val mlcLineDelim = Pattern.compile("\\h*+\n\\h*+[*]*+\\h*+")
+  private val mlcLineDelim = Pattern.compile("(?:\\h*+\n|^)\\h*+[*]*+\\h*+")
```

Comment:
Strange failure on CI. May be related to https://bugs.openjdk.java.net/browse/JDK-8189343

Reply:
how do you find those, @poslegm? you're likely right.

i found a different, slightly roundabout way to make it work.

Reply:
> how do you find those, @poslegm?

just googling java 8 vs java 9 regex difference 🤷